src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -12,10 +13,16 @@ namespace Microsoft.DotNet.Tools.Install.Tool
 {
     internal class ProjectRestorer : IProjectRestorer
     {
+        public ProjectRestorer(ICommandFactory commandFactory = null)
+        {
+            _commandFactory = commandFactory ?? new DotNetCommandFactory(alwaysRunOutOfProc: true);
+        }
+
         public void Restore(
             FilePath projectPath,
             DirectoryPath assetJsonOutput,
-            FilePath? nugetconfig)
+            FilePath? nugetconfig,
+            string source)
```

Comment:
this is a public method, and we should not expect null to be passed in public method. At least we should `string source = null`

Reply:
I'll fix.